src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -168,29 +172,24 @@ class HuiEntitiesCard extends LitElement implements LovelaceCard {
     if (this._hass) {
       element.hass = this._hass;
     }
+
     if (
-      entityConf.entity &&
-      !DOMAINS_HIDE_MORE_INFO.includes(computeDomain(entityConf.entity))
+      ((!entityConf.tap_action ||
+        entityConf.tap_action.action === "more-info") &&
+        !DOMAINS_HIDE_MORE_INFO.includes(computeDomain(entityConf.entity))) ||
```

Comment:
We shouldn't be smart here. If a user picks `more-info`, we are going to add `more-info`. Only use the `DOMAINS_HIDE_MORE_INFO` if a user did not specify a tap action.